Cargo.toml

```diff
@@ -51,6 +51,7 @@ lazy_static = "1.4.0"
 const-str = "0.3.1"
 pathdiff = "0.2.1"
 # CLI deps
+atty = "0.2"
```

Comment:
Can you make this dep optional so it's only pulled in for the CLI?

Reply:
Oops, missed that!